### PR TITLE
Maps API

### PR DESCRIPTION
**Pull request by critical-path**
This attempts to address #228.  I would be happy to make changes to the code, tests, etc.  Thank you!

### CODE REVIEW

File: docs/maps.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+Instantiate `Maps`.
+
+```python
+>>> maps = Maps()
+```
+
+Call the `get_tile` method, passing in values for `map_id`, `z` (zoom), `x` (column), and `y` (row).  Pass in values for optional arguments as necessary - `retina` (double scale), `file_format`, `style_id`, and `timestamp`.
+
+```python
+>>> response = maps.get_tile("mapbox.streets", z=0, x=0, y=0)
```
Comment (sgillies) [inline docs/maps.md]: @critical-path @perrygeo I think I'd prefer the tile params to be positional arguments in x, y, z order so we can pass in mercantile tiles like

`get_tile(mapid, *Tile(0, 0, 0))`

"get" is implied in the method, so how about just `tile()` ?

File: docs/maps.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+Instantiate `Maps`.
+
+```python
+>>> maps = Maps()
+```
+
+Call the `get_html_slippy_map` method, passing in a value for `map_id`.  Pass in values for optional arguments as necessary - `options` (map controls and behaviors), `z`, `lat`, and `lon`.
+
+```python
+>>> response = maps.get_html_slippy_map("mapbox.streets")
```
Comment (sgillies) [inline docs/maps.md]: I'm not certain about the usefulness of getting a slippy map in Python. I think we could eliminate this method and have less API to support.

**Comment by critical-path** [comment]
I have made all but one of the requested changes.  In the features method, I will wait for confirmation before removing the option to request either GeoJSON or KML format.  Thank you!

**Comment by perrygeo** [comment]
I'm inclined to keep the KML format option. I don't imagine too many people will use it but it's consistent with our design goal of keeping the mapbox-sdk-py as an un-opinionated, lightweight wrapper over the HTTP requests.

**Comment by critical-path** [comment]
I have replaced all instances of "standalone_marker" with "marker" in the code, tests, and docs.  Thank you!

**Pull request merged**